src/alto-ui/Datagrid/components/DatagridCell/DatagridCell.js

```diff
@@ -351,15 +351,16 @@ class DatagridCell extends React.Component {
   }
 
   render() {
-    const { aria } = this.props;
+    const { aria, column, row, context } = this.props;
     const style = this.getStyle();
     const modifiers = this.getModifiers();
     const value = this.getFormattedValue();
+    const isCellVisible = context.visible(column, row);
```

Comment:
It will be good to extract this logic in useMemo when DatagridCell will be refactor